base/workflow/R/runModule.run.write.configs.R

```diff
@@ -5,14 +5,19 @@
 #' @return A modified settings object, invisibly
 #' @importFrom dplyr %>%
 #' @export
-runModule.run.write.configs <- function(settings, overwrite = TRUE) {
+runModule.run.write.configs <- function(settings, overwrite = TRUE,input_design=NULL) {
+
 
   if (PEcAn.settings::is.MultiSettings(settings)) {
     if (overwrite && file.exists(file.path(settings$rundir, "runs.txt"))) {
       PEcAn.logger::logger.warn("Existing runs.txt file will be removed.")
       unlink(file.path(settings$rundir, "runs.txt"))
     }
-    return(PEcAn.settings::papply(settings, runModule.run.write.configs, overwrite = FALSE))
+    if(is.null(input_design)){
```

Comment:
Tidyverse style calls for spaces around the parens of an if statement
(This is one we're very inconsistent about in the existing codebase! But it's still worth improving)

```suggestion
    if (is.null(input_design)) {
```